### PR TITLE
[DOC canary] Document {{#if}} and {{#unless}} helpers

### PR DESCRIPTION
**Pull request by joeruello**
Noticed there were no docs for `if` and `unless`. Tried to write something up.

Let me know if there's anything wrong/I should change.

### CODE REVIEW

**Comment by joeruello** [comment]
Test seems to have failed on phantomjs, not sure how to diagnose and fix?

File: packages/ember-htmlbars/lib/helpers/if_unless.js [hunk 1]
```diff
@@ -7,6 +7,42 @@ import Ember from "ember-metal/core"; // Ember.assert
...
+    Good morning
+  {{else if isAfternoon}}
+    Good afternoon
+  {{else}}
+    Good night
+  {{/if}}
+  ```
+  You can use `if` inline to conditionally render a single property or string.
+  ```handlebars
+  {{if useLongGreeting "Hello" "Hi"}} Dave
```
Comment (mixonic) [inline packages/ember-htmlbars/lib/helpers/if_unless.js]: should probably clarify that this is a ternary, and what the three arguments are.

**Comment by mixonic** [comment]
@joeruello can you just add some whitespace ala these docs?

https://github.com/emberjs/ember.js/blob/master/packages/ember-htmlbars/lib/helpers/bind-attr.js#L7

We try to keep the length of a line around 80 characters, and put a newline between the text and code so things are easy to read in the source.

**Comment by joeruello** [comment]
@mixonic Done :)

**Comment by mixonic** [comment]
restarting the build

**Comment by rwjblue** [comment]
Thank you!

**Pull request merged**